ScreenToGif/Resources/Localization/StringResources.da.xaml

```diff
@@ -1021,26 +1227,33 @@
     <s:String x:Key="S.Recent.Date">Dato</s:String>
     <s:String x:Key="S.Recent.Frames">Rammeantal</s:String>
     <s:String x:Key="S.Recent.Warning.NoSelection">Intet valgt. Vælg et projekt fra listen.</s:String>
-    <!--<s:String x:Key="S.Recent.Warning.SameProject">This project is already being displayed by this editor instance.</s:String>-->
+    <s:String x:Key="S.Recent.Warning.SameProject">Projektet vises allerede.</s:String>
     <s:String x:Key="S.Recent.Warning.AnotherEditor">Projektet er allerede åbnet i redigering. Projekter kan kun åbnes i en programinstans.</s:String>
     
     <!--Editor • Clipboard-->
     <s:String x:Key="S.Clipboard.Entries">Udklipsholder</s:String>
     <s:String x:Key="S.Clipboard.Entry">Emne:</s:String>
-    <!--<s:String x:Key="S.Clipboard.Entry.Image">{0} image</s:String>-->
-    <!--<s:String x:Key="S.Clipboard.Entry.Images">{0} images</s:String>-->
+    <s:String x:Key="S.Clipboard.Entry.Image">{0} billede</s:String>
+    <s:String x:Key="S.Clipboard.Entry.Images">{0} billeder</s:String>
     <s:String x:Key="S.Clipboard.Explore">Gennemse...</s:String>
     <s:String x:Key="S.Clipboard.Remove">Fjern</s:String>
     <s:String x:Key="S.Clipboard.Behavior">Indsættelse</s:String>
     <s:String x:Key="S.Clipboard.Before">Før valgte ramme</s:String>
     <s:String x:Key="S.Clipboard.After">Efter valgte ramme</s:String>
     
     <!--Editor • Resize-->
-    <s:String x:Key="S.Resize.CurrentProperties">Egenskaber</s:String>
+    <s:String x:Key="S.Resize.Difference">Forskel</s:String>
     <s:String x:Key="S.Resize.Dpi">Dpi</s:String>
-    <s:String x:Key="S.Resize.NewProperties">Nye egenskaber</s:String>
+    <s:String x:Key="S.Resize.Options">Indstillinger</s:String>
+    <s:String x:Key="S.Resize.Pixels">Pixel (px)</s:String>
+    <s:String x:Key="S.Resize.Percent">Procent (%)</s:String>
     <s:String x:Key="S.Resize.Dpi2">DPI:</s:String>
     <s:String x:Key="S.Resize.KeepAspect">Bevar størrelsesforhold.</s:String>
+    -<s:String x:Key="S.Resize.ScalingQuality">Kvalitet:</s:String>
```

Comment:
Here are the lines with issue. Just erase the "-" in lines 1252-1256 or I'll do it for you when merging, :)

Reply:
Hi Nicke.
I've uploaded a new version to Github. It should be OK, but I still can't test it.
Now I get a whole different kind of error ([image](https://i.imgur.com/gb3uEUb.png)).
If you can use it, please do so, and I will take a look at it, when you publish the next STF update.
Kind regards 
Regmos

Reply:
Thanks. I'll fix that one.

Reply:
It was just an issue with the key `S.Recorder.Fps.Range`, there's other two keys with a similar name (`Fpm` and `Fph`) and the three were set as `Fps`.

Reply:
Hi Nicki.
Thats it. Now everything should be OK
I've reviewed it, fixed a few things, and uploaded a new version.
Kind regards
Regmos